geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/Makefile

```
@@ -34,3 +35,34 @@ eslint: ## Runs the eslint checks
 		eslint $(find {{cookiecutter.package}} -type f -name '*.js' -print 2> /dev/null)
 	docker-compose run --entrypoint= --no-deps --rm --volume=$(pwd)/geoportal:/app geoportal \
 		eslint $(find {{cookiecutter.package}} -type f -name '*.ts' -print 2> /dev/null)
+
+.PHONY: qgis
+qgis: ## Run QGIS desktop
+	docker-compose -f docker-compose.yaml -f docker-compose-qgis.yaml run --rm qgis
+
+secrets.tar.bz2.gpg: env.secrets ## Create the secrets.tar.bz2.gpg files we depends on
```

```suggestion
secrets.tar.bz2.gpg: env.secrets ## Encrypt the secrets for committing changes
```
Note that the filename is already the target name, not sure we need to repeat it in description.